docs/database/readme.md

```diff
@@ -0,0 +1,159 @@
+# Schema
+
+```mermaid
+---
+title: Airneis database model
+---
+
+classDiagram
+
+Categories <|-- Products
+Carts <|-- Status
+Products <|-- Carts
+Users <|-- Addresses
+Users <|-- Carts
+Carts <|-- CartsProducts
+Products <|-- CartsProducts
+Products <|-- ProductImages
+Materials <|-- ProductsMaterials
+Products <|-- ProductsMaterials
+
+class Users {
```

Comment:
Why do we use `fullname` in the users table and `firstName text notNull, lastName text notNull` in the addresses table. Choose one